github/git_blobs.go

```diff
@@ -38,6 +39,23 @@ func (s *GitService) GetBlob(ctx context.Context, owner string, repo string, sha
 	return blob, resp, err
 }
 
+// GetBlobRaw fetches a blob's contents from a repo.
+// Unlike GetBlob, it returns the raw bytes rather than the base64-encoded data.
```

Comment:
Please add a GitHub API link. It's the same as the one above:

```
//
// GitHub API docs: https://developer.github.com/v3/git/blobs/#get-a-blob
```

Reply:
done